src/scss/constants.scss

```diff
@@ -271,30 +271,19 @@ $z-layers: (
   }
 }
 
-// @debt use only one scrollbar mixin (if two different styles are needed, add variables)
 @mixin scrollbar {
-  &::-webkit-scrollbar-track {
-    border-radius: 3px;
-    background-color: rgba(0, 0, 0, 0);
-  }
-
-  &::-webkit-scrollbar {
-    width: 8px;
-    height: 8px;
-    background-color: rgba($black, 0);
-  }
-
-  &::-webkit-scrollbar-thumb {
-    border-radius: 4px;
-    background-color: rgba($white, 0.3);
-  }
-
-  &::-webkit-scrollbar-thumb:hover {
-    background-color: rgba($white, 0.5);
+  // The standard properties
+  // @see https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-color
+  // @see https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width
+  scrollbar-color: opaque-white(0.3) $black;
+  scrollbar-width: thin;
```

Comment:
is it a good idea to use them? Because it seems that it's supported only in 3% browsers now..

https://caniuse.com/?search=scrollbar-color

Reply:
They make a difference in Firefox, so as long as other browsers ignore them - good. If webkit ones decide to support them - good again, it's the same colors.

Reply:
The support so far https://caniuse.com/?search=scrollbar-

Reply:
your solution is supported by 3,16 % of the browser
and the solution you've removed is supported by 92,02% (see https://caniuse.com/?search=webkit-scrollbar)

We could use your solution as an addition to the existing one, but to remove the old one - not sure it was a good idea.

Reply:
> your solution is supported by 3,16 % of the browser
and the solution you've removed is supported by 92,02% (

@sunny-viktoryia  what solution did I remove?

Reply:
sorry, @goran-peoski-work, looks that I was wrong. I thought that you've just removed the old solution with `::-webkit-scrollbar`, but in fact you've just removed the duplicate.